client/ionic/src/pages/About.tsx

```diff
@@ -11,111 +12,83 @@ import {
-                  from person-to-person between people who are in close contact
-                  with one another (within about 6 feet or two meters) through
-                  respiratory droplets produced when an infected person coughs
-                  or sneezes.
-                </IonCardSubtitle>
-              </IonCardHeader>
-              <IonCardContent>
-                <IonIcon size="large" icon={personAddOutline}></IonIcon>
-              </IonCardContent>
-              <IonCardContent>
-                <IonButton href="/menu" shape="round">
-                  Learn More
-                </IonButton>
-              </IonCardContent>
-            </IonCard>
-          </IonSlide>
-          <IonSlide>
-            <IonCard>
-              <IonCardHeader>
-                <IonCardTitle>Clean Your Hands</IonCardTitle>
-                <IonCardSubtitle>
-                  <b>Wash your hands often</b> with soad and water for at least
-                  20 sconds, especially after you have been in a public place,
-                  or after blowing your nose, coughing or sneezing.
-                </IonCardSubtitle>
-                <IonCardSubtitle>
-                  <b>Avoid touching your eyes, nose, and mouth</b> with unwashed
-                  hands.
-                </IonCardSubtitle>
-              </IonCardHeader>
-              <IonCardContent>
-                <IonIcon size="large" icon={handLeftOutline}></IonIcon>
-                <IonIcon size="large" icon={handRightOutline}></IonIcon>
-              </IonCardContent>
-              <IonCardContent>
-                <IonButton href="/menu" shape="round">
-                  Learn More
-                </IonButton>
-              </IonCardContent>
-            </IonCard>
-          </IonSlide>
-          <IonSlide>
-            <IonCard>
-              <IonCardHeader>
-                <IonCardTitle>Avoid Close Contact</IonCardTitle>
-                <IonCardSubtitle>
-                  <b>Avoid close contact</b> with people who are sick.
-                </IonCardSubtitle>
-                <IonCardSubtitle>
-                  <b>Maintain distance between yourself and other people</b> if
-                  COVID-19 is spreading in your community. This is especially
-                  important for people who are at a higher risk of getting very
-                  sick, including the elderly.
-                </IonCardSubtitle>
-              </IonCardHeader>
-              <IonCardContent>
-                <IonIcon size="large" icon={peopleOutline}></IonIcon>
-              </IonCardContent>
-              <IonCardContent>
-                <IonButton href="/menu" shape="round">
-                  Learn More
-                </IonButton>
-              </IonCardContent>
-            </IonCard>
-          </IonSlide>
-        </IonSlides>
+        {flow.content && flow.content.screens && (
+          <IonSlides pager={true}>
```

Comment:
~are screens in a flow only intended to be slides in an IonSlides component, or can they also represent full page screens in the app?~ found my own answer [here](https://github.com/WorldHealthOrganization/app/pull/120/files#diff-d86ea463733dce0c82a6e878dbd9319cR38)